src/app/components/policy-guide/docs/compliance/compliance-inventory-code/compliance-inventory-code.template.html

```diff
@@ -46,7 +46,7 @@ <h4>Required</h4>
     <li><code>ifOther</code>: [string] A one- or two- sentence description of the measurement type used, if 'other' is selected as the value of 'method' field.</li>
 			</ul>
   </li>
-	<li><code>agency</code>: [string] The agency acronym for Clinger Cohen Act agency, as defined by the United States Government Manual. For example "GSA" or "DOD." Please use uppercase letters.</li>
+	<li><code>agency</code>: [string] The agency acronym for <a href="https://cfo.gov/about/">Clinger Cohen Act agency</a>, as defined by the United States Government Manual. For example "GSA" or "DOD." Please use uppercase letters.</li>
```

Comment:
I would want to change this to say "The agency acronym for Clinger Cohen Act agency can be found through the CFO Act list, as CCA references the CFO Act. For example 'GSA' or 'DOD.' Please..."

Reply:
Updated the wording.